MantidQt/CustomInterfaces/test/EnggDiffFittingPresenterTest.h

```diff
@@ -361,7 +366,7 @@ class EnggDiffFittingPresenterTest : public CxxTest::TestSuite {
 
     // No errors/1 warnings. File entered is not found
     EXPECT_CALL(mockView, userError(testing::_, testing::_)).Times(0);
-    EXPECT_CALL(mockView, userWarning(testing::_, testing::_)).Times(1);
+    EXPECT_CALL(mockView, userWarning(testing::_, testing::_)).Times(2);
 
     pres.notify(IEnggDiffFittingPresenter::FittingRunNo);
```

Comment:
I'll add this to #17433 as there are several unit tests which currently do not call `VerifyAndClearExpectations`